### PR TITLE
feat: added wishlisht functionality

### PR DESCRIPTION
**Pull request by AnjaliDeshwani**
**This PR contains:**

- Add to Wishlist 
- Remove from Wishlist 
- View and remove the products on the wishlist list

**Preview available at -** https://deploy-preview-3--winsome-books-store.netlify.app/product-list

### CODE REVIEW

File: src/pages/Wishlist/Wishlist.jsx [hunk 1]
```diff
@@ -1,76 +1,79 @@
...
+                          }
+                        >
+                          <span className="delete-icon">
+                            <i className="fa-solid fa-trash"></i>
+                          </span>
+                        </div>
+                      </div>
+                    </div>
+                    <div class="border-bottom"></div>
+                  </div>
```
Comment (JowelTisso) [inline src/pages/Wishlist/Wishlist.jsx]: Typo here

**Review by JowelTisso — APPROVED** [review]
Good Constant naming
Rest looks good

**Review by shobit1337 — APPROVED** [review]
Your code looks clean and well strucured. Good work @AnjaliDeshwani

**Review by Nikhil-Nirmal — APPROVED** [review]
Nice and clean code.

File: src/pages/Wishlist/Wishlist.jsx (hunk 1 above)
Comment (AnjaliDeshwani) [reply src/pages/Wishlist/Wishlist.jsx]: I have corrected it now, thanks Jowel for catching it!

**Pull request merged**